utils.py

```diff
@@ -13,7 +13,7 @@
 import os
 from gi.repository import GdkPixbuf
 import subprocess
-from StringIO import StringIO
+from io import StringIO
 import json
 json.dumps
 from json import load as jload
```

Comment:
On line (90) if you probably get time, consider using base64 builtin python3 module. 
```
import base64
data = base64.b64decode(string)
```
This will reduce the code at lines 86-95

Reply:
Sorry, what do you mean by that?